obstacle_stop_planner_refine/include/obstacle_stop_planner/node.hpp

```diff
@@ -20,56 +20,85 @@
+    if (it != parameters.cend()) {
+      value = it->template get_value<T>();
+    }
+  }
+}
+
 namespace obstacle_stop_planner
 {
+using PointCloud2 = sensor_msgs::msg::PointCloud2;
+using Trajectory = autoware_planning_msgs::msg::Trajectory;
+using TwistStamped = geometry_msgs::msg::TwistStamped;
+using DynamicObjectArray = autoware_perception_msgs::msg::DynamicObjectArray;
+using MarkerArray = visualization_msgs::msg::MarkerArray;
+using StopReasonArray = autoware_planning_msgs::msg::StopReasonArray;
+using Float32MultiArrayStamped = autoware_debug_msgs::msg::Float32MultiArrayStamped;
+using Point3d = autoware_utils::Point3d;
+
 class OBSTACLE_STOP_PLANNER_PUBLIC ObstacleStopPlannerNode : public rclcpp::Node
 {
 public:
   explicit ObstacleStopPlannerNode(const rclcpp::NodeOptions & options);
 
 private:
-  /*
-   * ROS
-   */
-  // publisher and subscriber
-  rclcpp::Subscription<autoware_planning_msgs::msg::Trajectory>::SharedPtr path_sub_;
-  rclcpp::Subscription<sensor_msgs::msg::PointCloud2>::SharedPtr obstacle_pointcloud_sub_;
-  rclcpp::Subscription<geometry_msgs::msg::TwistStamped>::SharedPtr current_velocity_sub_;
-  rclcpp::Subscription<autoware_perception_msgs::msg::DynamicObjectArray>::SharedPtr
-    dynamic_object_sub_;
-  rclcpp::Subscription<autoware_planning_msgs::msg::ExpandStopRange>::SharedPtr
-    expand_stop_range_sub_;
-  rclcpp::Publisher<autoware_planning_msgs::msg::Trajectory>::SharedPtr path_pub_;
-
-  tf2_ros::Buffer tf_buffer_;
-  tf2_ros::TransformListener tf_listener_;
-  sensor_msgs::msg::PointCloud2::ConstSharedPtr obstacle_pointcloud_ptr_;
-
-  /*
-   * Parameter
-   */
-  geometry_msgs::msg::TwistStamped::ConstSharedPtr current_velocity_ptr_;
-  autoware_perception_msgs::msg::DynamicObjectArray::ConstSharedPtr object_ptr_;
-  rclcpp::Time prev_col_point_time_;
-  pcl::PointXYZ prev_col_point_;
-  bool pointcloud_received_;
-  bool current_velocity_reveived_;
+  // Publisher
+  rclcpp::Publisher<Trajectory>::SharedPtr pub_path_;
+  rclcpp::Publisher<MarkerArray>::SharedPtr pub_debug_viz_;
+  rclcpp::Publisher<StopReasonArray>::SharedPtr pub_stop_reason_;
+  rclcpp::Publisher<Float32MultiArrayStamped>::SharedPtr pub_acc_debug_;
+
+  // Subscriber
+  rclcpp::Subscription<Trajectory>::SharedPtr sub_path_;
+  rclcpp::Subscription<PointCloud2>::SharedPtr sub_obstacle_pointcloud_;
+  rclcpp::Subscription<TwistStamped>::SharedPtr sub_current_velocity_;
+  rclcpp::Subscription<DynamicObjectArray>::SharedPtr sub_dynamic_object_;
+
+  // Parameter callback
+  OnSetParametersCallbackHandle::SharedPtr set_param_res_;
+  rcl_interfaces::msg::SetParametersResult onParameter(
+  const std::vector<rclcpp::Parameter> & parameters);
+
+  // Parameter
+  StopControlParameter stop_param_;
+  SlowDownControlParameter slow_down_param_;
+  AdaptiveCruiseControlParameter acc_param_;
+
+  autoware_utils::SelfPoseListener self_pose_listener_;
+  TransformListener transform_listener_;
```

Comment:
Is it possible to write `self_pose_listener_{this}` or not?

Reply:
Yes, it is possible.
The Google C ++ style guide didn't say which is the recommended method, but at least this package needs to be written in the same way to avoid confusion.

Reply:
What do you mean be `the same way`?